<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widgetMenu{
background-color:rgb(175, 189, 212);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetMenu" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>25</y>
      <width>180</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QSlider" name="SliderSizeOfShip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SliderSizeOfWay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="pageStep">
      <number>3</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="OnVisibleAndWayBut">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>On Way and Visible</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OffVisibleAndWayBut">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Off Way and Visible</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size Of Ship</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size Of Way</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="TextInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>161</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="MenuBut">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
